internal/handler/handler.go

```diff
@@ -55,24 +130,16 @@ func (svc userMgmtSvc) HealthCheck() (svcName string, msg string, stat bool) {
 	return
 }
 func (svc userMgmtSvc) SignUp(w http.ResponseWriter, r *http.Request) {
-
 	var credential model.SignUpCredentials
-	body, err := ioutil.ReadAll(r.Body)
+	status, err := request.FromJson(r, &credential)
 	if err != nil {
 		log.Error(err)
-		response.ToJson(w, http.StatusBadRequest, codes.GetErr(codes.ErrReadingReqBody), nil)
+		response.ToJson(w, status, err.Error(), nil)
```

Comment:
All error messages must follow the error coded format. The error returned from the `FromJson()` is a generic error. 
Earlier error message with code can be put back for this case.
```go
codes.GetErr(codes.ErrReadingReqBody)
```

Implement for other responses.

Reply:
done

Reply:
Still seeing codes not used.

Reply:
inside response.tojson func the error returned is already returning specific error with error codes